### PR TITLE
Tweaks to the walkthrough for local-up-cluster

### PR DESCRIPTION
**Pull request by pmorie**
A couple tweaks were necessary for me to make the walkthrough run on a cluster created with `local-up-cluster.sh`:

- Needed to make the default service account for the `kube-system` namespace a cluster admin
- Needed a knob to turn off TLS skip verify for the controller -> SC API server connection (you can't set insecure skip verify AND provide root cert data at the same time)

### CODE REVIEW

File: docs/walkthrough.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ DNS enabled already.
 * If you are using hack/local-up-cluster.sh, ensure the
 `KUBE_ENABLE_CLUSTER_DNS` environment variable is set as follows:
 
-  ```console
-  KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O
-  ```
+```console
+KUBE_ENABLE_CLUSTER_DNS=true hack/local-up-cluster.sh -O
```
Comment (sjenning) [inline docs/walkthrough.md]: `KUBE_ENABLE_CLUSTER_DNS=true` is the default in kube now

File: charts/catalog/templates/controller-manager-deployment.yaml [hunk 2]
```diff
@@ -55,7 +55,9 @@ spec:
         - --service-catalog-api-server-url
         - https://{{ template "fullname" . }}-apiserver
         {{- end }}
+        {{ if .Values.controllerManager.apiserverSkipVerify -}}
         - "--service-catalog-insecure-skip-verify=true"
+        {{- end }}
```
Comment (MHBauer) [inline charts/catalog/templates/controller-manager-deployment.yaml]: Overall question:
Do we need an issue to make sure every single flag is helm exposed?

File: docs/walkthrough.md [hunk 3]
```diff
@@ -32,6 +32,18 @@ be done with Helm setup.
...
 [installation instructions](https://github.com/kubernetes/helm/blob/master/docs/install.md).
 
+### RBAC Considerations
+
+If your kubernetes cluster has [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/)
+enabled, you must ensure that the default service account for the `kube-system`
+namespace has the `cluster-admin` role:
+
+```console
+kubectl create clusterrolebinding default-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
```
Comment (MHBauer) [inline docs/walkthrough.md]: kubeadm sets up this level of access, and while I have to include the disclaimer that kubeadm is not production ready, I have to somewhat assume that a decision was made that this is the correct level of access. 

An additional data point is that Bluemix does include this serviceAccount bound to this level of access.

File: charts/catalog/templates/controller-manager-deployment.yaml (hunk 2 above)
Comment (pmorie) [inline charts/catalog/templates/controller-manager-deployment.yaml]: #1083

**Pull request merged**